src/appConstants/index.ts

```diff
@@ -19,6 +20,9 @@ export const SET_TEAM_PASSWORD = 'SET_TEAM_PASSWORD';
 export const SET_SOCIAL_LINK = 'SET_SOCIAL_LINK';
 export const SET_FILTER_DATA = 'SET_FILTER_DATA';
 export const SET_CURR_LANG = 'SET_CURR_LANG';
+export const SET_COURSE_SELECT_OPEN = 'SET_COURSE_SELECT_OPEN';
+export const SET_HEADER_LANG_SELECT_OPEN = 'SET_HEADER_LANG_SELECT_OPEN';
```

Comment:
Is it worth to store the selects open state in redux? Is the info used somewhere else in the app and required for any functionality?

Reply:
Done